components/tag-connect/common.stanza

```diff
@@ -0,0 +1,60 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tag-connect/common : 
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/bundles
+
+public defn make-10-pin-connector-module (component:Instantiable) : 
+  inside pcb-module : 
+    ; the connecting component
+    inst conn: component
+
+    ; the bundles we support out of the module
+    val power-bundle  = power
+    val reset-bundle  = reset
+    val swd-bundle    = swd([SWD-SWO, SWD-TRACESWO]) 
+    val jtag-bundle = jtag()
+
+    port jtag  : jtag-bundle
+    port swd   : swd-bundle
+    port power : power-bundle
+    port reset : reset-bundle
+
+    ; hook up power
+    net (power.vdd, conn.p[1])
+    net (power.gnd, conn.p[3], conn.p[5], conn.p[7], conn.p[9])s
+    supports power-bundle : 
+      power-bundle.vdd => power.vdd
+      power-bundle.gnd => power.gnd
+    
+    ; hook up jtag
+    net (jtag.tms, conn.p[2])
+    net (jtag.tck, conn.p[4])
+    net (jtag.tdo, conn.p[6])
+    net (jtag.tdi, conn.p[8])
+    net (jtag.trstn conn.p[10])
+    
+    supports jtag-bundle : 
+      jtag-bundle.tms => jtag.tms
+      jtag-bundle.tck => jtag.tck
+      jtag-bundle.tdo => jtag.tdo
+      jtag-bundle.tdi => jtag.tdi
+      jtag-bundle.trstn => jtag.trstn
+        
+    ; Note: this will not be used if JTAG is used with trstn
+    net (reset.reset, conn.p[10])
+    supports reset-bundle : 
+      reset.reset => reset-bundle.reset
```

Comment:
I just built off the dev branch, and get this error:

[Error] Invalid reference to invalid definition.
  at jitpcb/src/utils/plugins.stanza:134.8
  at core/core.stanza:4853.4
  at core/core.stanza:4843.3
  at jitpcb/src/utils/plugins-master.stanza:28.9
  at core/core.stanza:4863.28
  at jitpcb/src/utils/plugins-master.stanza:27.4
  at jitpcb/src/jitpcb/esir/client-plugin.stanza:114.2
  at jitpcb/src/jitpcb/esir/client.stanza:240.6
  at open-components-database/components/tag-connect/common.stanza:49.6
...

Reply:
Each mapping inside a support should look this:
`<bundle pin> =>  <local / required pin>`

`reset-bundle.reset` => `reset.reset`
should be correct.

Reply:
Thanks, fixed!